script.js

```diff
@@ -1,15 +1,106 @@
-const modal = document.querySelector('#instructionModal');
-const closeBtn = document.querySelector('.close');
+const instructionModal = document.querySelector('#instructionModal');
+const instructionCloseBtn = document.querySelector('.instructionClose');
 const instructionsButton = document.querySelector('#instructionsButton')
+const startButton = document.querySelector('.start')
+const gameOverModal = document.querySelector('#gameOverModal');
+const gameOverCloseBtn = document.querySelector('.gameOverClose');
 
-closeBtn.addEventListener('click', () => {
-    modal.classList.remove('open');
-});
 
 
-const openInstructionsModal = () => {
+const openModal = (modal) => {
     modal.classList.add('open');
 }
 
-instructionsButton.addEventListener('click', openInstructionsModal);
+const closeModal = (modal) => {
+    modal.classList.remove('open');
+}
+
+instructionsButton.addEventListener('click', () => {
+    openModal(instructionModal)
+})
+
+instructionCloseBtn.addEventListener('click', () => {
+    closeModal(instructionModal);
+})
+
+gameOverCloseBtn.addEventListener('click', () => {
+    closeModal(gameOverModal);
+})
+
+const gameOver = () => {
+    openModal(gameOverModal)
+}
+// max's get pattern functions:
+
+// const
+let pattern = []
+function getRandBoxes(boxCount) {
```

Comment:
why have you changed to using a name function when all the others are constants with fat arrow functions?